apps/api_web/lib/api_web/views/stop_view.ex

```diff
@@ -83,6 +89,10 @@ defmodule ApiWeb.StopView do
     State.Transfer.recommended_transfers_from(stop_id)
   end
 
+  def zone(%{id: stop_id}, _conn) do
+    State.Stop.zone_id(stop_id)
```

Comment:
I think the first argument here is a `%Stop{}` already, so you could match it out:
```elixir
def zone(%{zone_id: zone_id}, _conn), do: zone_id
```